quantities.serialization/newtonsoft/Extensions.cs

```diff
@@ -0,0 +1,12 @@
+using Newtonsoft.Json;
+
+namespace Quantities.Serialization.Newtonsoft;
+
+public static class Extensions
+{
+    public static JsonSerializerSettings EnableQuantities(this JsonSerializerSettings settings)
```

Comment:
The  respective `EnableQuantities` extension method for Newtonsoft.

Note: all other types are (at most) internal...